test/hotspot/jtreg/containers/docker/TestMisc.java

```diff
@@ -86,7 +86,7 @@ private static void testIsContainerized() throws Exception {
     private static void testPrintContainerInfo() throws Exception {
         Common.logNewTestCase("Test print_container_info()");
 
-        DockerRunOptions opts = Common.newOpts(imageName, "PrintContainerInfo");
+        DockerRunOptions opts = Common.newOpts(imageName, "PrintContainerInfo", "-XX:ActiveProcessorCount=2");
```

Comment:
It would be better to separate this specific test. More like this:
https://github.com/jerboaa/jdk/commit/3911ef81c8f280a2c0da259211b91f4c3e8c3994

Reply:
I'm not sure I like this version of the test better. I'd rather separate concerns. The changes to `Common` aren't really needed, as `Common.newOpts(imageName, "PrintContainerInfo").addJavaOpts("-XX:ActiveProcessorCount=2");` can be used instead. Similarly, the changes to `checkContainerInfo()` make the test harder to read. It's once called with a `null` argument and checks similar things, but one additional assert. Rather than that, I'd just do the assert directly in the new test as suggested. This saves the reader having to look up call-sites to know what the test is actually asserting. This also more closely matches what's likely to happen in real scenarios (override the container detection heuristics with `-XX:ActiveProcessorCount`).